scripts/r.in.wms/wms_drv.py

```diff
@@ -18,8 +18,16 @@
 
 import socket
 import grass.script as grass
+import numpy as Numeric
```

Comment:
I know it was there before, but probably a new PR for this, but it is almost an universal convention across all Python code that numpy is imported as np, like `import numpy as np`. Why should we be different here?

Probably someone from this repo can jump in.

Reply:
It was added a long time ago in https://github.com/OSGeo/grass/commit/31d721a2b3872df59e823dd03750c45932778d9a - perhaps the style of back then? Can @landam remember?

Reply:
I think it is just some very old practice carried over. Just change that.